ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -164,6 +168,16 @@ private static void setTransformMatrix(View view, ReadableMap matrix) {
         (float) matrix.getDouble(PROP_DECOMPOSED_MATRIX_SCALE_X));
     view.setScaleY(
         (float) matrix.getDouble(PROP_DECOMPOSED_MATRIX_SCALE_Y));
+        
+    if(matrix.hasKey(PROP_DECOMPOSED_MATRIX_PERSPECTIVE)) {
+      ReadableArray perspectiveArray = matrix.getArray(PROP_DECOMPOSED_MATRIX_PERSPECTIVE);
+      if(perspectiveArray.size() == 4) {
+        float perspective = (float)(-1 / perspectiveArray.getDouble(2));
```

Comment:
Is is possible to use one of the constants to get the value from the array instead of using getDouble(2) so it is clear what this value represents?